extensions/frontend-tiles/src/integration-test/IntegrationTest.test.ts

```diff
@@ -0,0 +1,108 @@
+import { IModelApp } from "@itwin/core-frontend";
+import { expect } from "chai";
+import { TestContext } from "./TestContext";
+import { GraphicRepresentationStatus } from "../GraphicsProvider/GraphicRepresentationProvider";
+
+async function getExport(exportId: string, accessToken: string, urlPrefix?: string): Promise<any> {
+  const headers = {
+    /* eslint-disable-next-line @typescript-eslint/naming-convention */
+    Authorization: accessToken,
+    /* eslint-disable-next-line @typescript-eslint/naming-convention */
+    Accept: "application/vnd.bentley.itwin-platform.v1+json",
+  };
+
+  // obtain export for specified export id
+  const url = `https://${urlPrefix}api.bentley.com/mesh-export/${exportId}`;
+  try {
+    const response = await fetch(url, { headers });
+    const result = await response.json();
+    return result;
+  } catch (err) {
+    return undefined;
+  }
+}
+
+describe("MeshExportServiceIntegrationTest", () => {
```

Comment:
dont use the global, import this

Reply:
I'm not sure what you mean by the global here, what should I be importing?

Reply:
im talking about `describe`, `before`, `after`, `it`, `test`, etc

Reply:
I see these being imported for tests running using vitest instead of certa, which currently this one is set up to run with. Should this be changed to run with vitest? Unless I'm missing it, I don't see other tests run with certa which import describe, etc.

Reply:
Due to current difficulty using the oidc-signin-tool with vitest, we've elected to keep the test running with certa. Because of that, I don't believe we need to import these things here, let me know if I'm mistaken on that.

Reply:
are they not exported from chai or mocha?

Reply:
They are exported from mocha, but I don't think it's necessary to import them in this case. I don't see any other tests which still use certa importing them - example [here](https://github.com/iTwin/itwinjs-core/blob/18403314367ce0879c609d308931823050f77c30/extensions/frontend-tiles/src/test/ModelGroup.test.ts#L82). Additionally, when I did import them from mocha, I was hit with webpack errors when trying to wrap them up.